lib/EarlyFunctionAnalysis/CFGAnalyzer.cpp

```diff
@@ -1001,8 +1012,34 @@ void CallSummarizer::handleCall(MetaAddress CallerBlock,
 
   Builder.CreateCall(PreCallHook, Args);
 
+  const auto SymbolName = extractFromConstantStringPtr(SymbolNamePointer);
+  const auto &[CalleeSummary, IsTail] = Oracle.getCallSite(Caller,
+                                                           CallerBlock,
+                                                           Callee,
+                                                           SymbolName);
+  const auto &ABIResults = CalleeSummary->ABIResults;
+
+  for (const auto &[Variable, Values] : ABIResults.ArgumentsRegisters) {
+    if (Values == abi::RegisterState::Yes) {
+      auto Register = M->getNamedGlobal(Variable->getName());
+      Builder.CreateLoad(Register);
```

Comment:
This load will be unused. Consider that if we run any optimization pipeline after this, this will be DCE'd away.